src/components/Message/component.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default ({ msg }) => {
+  const content = msg || 'You\'re here!';
```

Comment:
es6 default parametres + no strings to be stubbed. Constant messages to configuration json of the module